src/library/itunes/itunesfeature.cpp

```diff
@@ -713,6 +735,10 @@ void ITunesFeature::parsePlaylist(QXmlStreamReader &xml, QSqlQuery &query_insert
                 //qDebug() << "exit playlist";
                 break;
             }
+            if (xml.name() == "dict" && !isPlaylistItemsStarted){
+                // Some playlists can be empty, so we need to exit.
+                break;
```

Comment:
Why is this required? 
Without testing, I would say it has to continue to parse the remaining crates.